config/resque_schedule.yml

```diff
@@ -75,3 +75,12 @@ check_invite_queue:
   queue: utilities
   args: check_queue
   description: "Invite users from the queue if it's time to do so."
+
+cleanup_work_original_creators:
+  every: 1h
+  class: "WorkOriginalCreator"
+  queue: utilities
+  args: cleanup
+  description: >-
+    Remove original_creators for works orphaned/moved more than
+    ORIGINAL_CREATOR_TTL_HOURS hours ago.
```

Comment:
Somehow I got it in my head I'd tested this... but I definitely didn't. I've `include`d `AsyncWithResque` since I don't think we need anything custom in `perform`. How does it look?